cypress/integration/core-opensearch-dashboards/vanilla-opensearch-dashboards/datasource-management-plugin/1_create_datasource.spec.js

```diff
@@ -0,0 +1,159 @@
+import { MiscUtils } from '@opensearch-dashboards-test/opensearch-dashboards-test-library';
+
+const miscUtils = new MiscUtils(cy);
+// Get environment variables
+const endPointUrl = Cypress.env('OSD_TEST_DOMAIN_ENDPOINT_URL');
+const invalidEndPointUrl = Cypress.env('OSD_INVALID_ENPOINT_URL');
+const username = Cypress.env('username');
+const password = Cypress.env('password');
+
+describe('Create datasources', () => {
+  beforeEach(() => {
+    // Visit OSD create page
+    miscUtils.visitPage(
+      'app/management/opensearch-dashboards/dataSources/create'
```

Comment:
This page will not load because we need config to enable data source plugin. Let's figure out how to override the config.

Reply:
Solved.